app/src/main/java/ch/epfl/skysync/models/flight/FlightType.kt

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.skysync.models.flight
+
+/**
+ * represents a particular flight type
+ *
+ * @property name name of the type
+ * @property specialRoles roles in addition to the BASE_ROLES(crew, pilot)
+ */
+data class FlightType(val name: String, val specialRoles: List<RoleType> = emptyList()) {
```

Comment:
I forgot to mention it on the first review, but `FlightType` also needs an `id` field